qcodes/instrument/parameter.py

```diff
@@ -797,6 +796,16 @@ def inter_delay(self, inter_delay: float) -> None:
                 'inter_delay ({}) must not be negative'.format(inter_delay))
         self._inter_delay = inter_delay
 
+    @property
+    def name(self) -> str:
+        """Name of the parameter"""
```

Comment:
Should we clarify that this is identical to short name.

Reply:
Done